src/pages/user/index.tsx

```diff
@@ -1,5 +1,7 @@
+import InsertionDisplay from '../../components/Insertion/InsertionDisplay';
+
 const UserProfile: React.FC = () => {
-  return <h1>User Page</h1>;
+  return <InsertionDisplay />;
```

Comment:
ziu pira continui ad avere una pagina che renderizza un componente e basta 😭 io farei direttamente quello che devi fare qua dentro, specialmente il data-fetching

ripeto, mia take personale quindi puoi non dargli peso, pero e' stranino, circa

Reply:
Ti dico che in realtà a questo punto toglierei direttamente le pagine e utilizzerei solo i componenti, perché non me ne faccio nulla poh